src/System.Security.Cryptography.Xml/tests/DSAKeyValueTest.cs

```diff
@@ -123,5 +123,51 @@ public void LoadXml_Null()
             DSAKeyValue dsa1 = new DSAKeyValue();
             Assert.Throws<ArgumentNullException>(() => dsa1.LoadXml(null));
         }
+
+        [Fact]
+        public void ImportDSAKeyValue()
+        {
+            string p = "kkos0S7rZLGluCzOcQ/S9L79be47MxRp3kB07vkX+iin4daLcSo1pg/K9aP0UJzZAKUAyT8uj7jci6c5sfVWdqmlRqtm0mwnfh88JgmH/Eu5LkMrUAV2gDQ8Cs8ihd2UbbQ0s47P65qrINVpkHaNOFJbz8vcSP598kNLg73QKh0=";
+            string q = "l4A4DwlgltuK00dcNNW5Yw2SbcM=";
+            string g = "NiI1mc0jI7JQa21Pbxo2e87cJtUfO1kCor0LHbOk1T0W5h6uhIgdNwlmE/ARTqfcw+RamtHTEI9HfxTg1TCNPQnWYenEIOGaot0qRZs2a2WzD2xjictcEA2D8up0sqpK/qTcnNV9g6hcc3HAm9mU6DjQ21MOsIoYbQB1VW91FrA=";
+            string y = "Q2vVSH6O8/v6jJq4r3PQaCLKThnbRXxluEq9mEY0XnXTwefwTJ67fUcCxhJJhwh157gQN2LWmte5gnM11VJMWT07DLIGPb7he2wPRGmgnS4nkChGgJGKXss5unzFkX4m+BxCCXZrClrJyahlZWAJY5CbLanzHSZQtRHIDdINxf4=";
+            string seed = "zfYSHDy431BKUVSpA6KqGnCsbrE=";
+            string pgenCounter = "sg==";
```

Comment:
From memory, didn't we run into an edge case where pgenCounter was more than 1 byte long? Might be worth testing... :)